utils/debounce.js

```diff
@@ -0,0 +1,7 @@
+export default function debounce(callback, wait = 250) {
```

Comment:
Why do you need the `debounce`? Maybe it's worth to put it in place, it's temporary anyway 🙏

Reply:
I consider it a kind of mandatory function in JS projects. I bet we will use it here. :)

Reply:
I do not agree because I personally never needed it. However, I trust your opinion.